internal/encoding/encoder_test.go

```diff
@@ -0,0 +1,65 @@
+package encoding
+
+import (
+	"testing"
+
+	"cuelang.org/go/cue"
+	"cuelang.org/go/cue/build"
+	"cuelang.org/go/cue/parser"
+)
+
+func TestEncoderValidation(t *testing.T) {
```

Comment:
do we still need this test now we have the other test?

Reply:
agreed; that's what I meant originally. this file can be deleted.